src/App.js

```diff
@@ -10,6 +11,7 @@ function App() {
       <Routes>
         <Route path="/" element={<Header />} />
         <Route path="/articles" element={<Articles />} />
```

Comment:
Currently on the home page (on path "/") you are rendering the header twice and there is no link to view all the articles. If you make the AllArticles component render instead of the second header that should solve this. Also Remember you don't _have_ to match the api routes with the front end routes.